tests/DfE.FindInformationAcademiesTrusts.UnitTests/ApplicationInsightsOptionsTests.cs

```diff
@@ -0,0 +1,21 @@
+using DfE.FindInformationAcademiesTrusts.Options;
+
+namespace DfE.FindInformationAcademiesTrusts.UnitTests;
+
+public class ApplicationInsightsOptionsTests
+{
+    [Fact]
+    public void ApplicationInsightsOptions_Should_Return_Correct_Section_String()
+    {
+        var expectedConfigurationSection = "ApplicationInsights";
+        var sut = ApplicationInsightsOptions.ConfigurationSection;
+        sut.Should().BeEquivalentTo(expectedConfigurationSection);
```

Comment:
For string comparison in Fluent Assertions it's better to use `Be` than `BeEquivalentTo`
Also, given that we're just testing the value of a static property, we could simplify this test greatly to
```csharp
ApplicationInsightsOptions.ConfigurationSection.Should().Be("ApplicationInsights");
```
This makes it a lot more easily readable